InWorldz/InWorldz.Phlox.Engine/LSLSystemAPI.cs

```diff
@@ -4481,7 +4481,7 @@ public void llRequestPermissions(string agent, int perm)
 
             // Okay, now we need to ask the user for permission.
             string ownerName = resolveName(m_host.ParentGroup.RootPart.OwnerID);
-            if (ownerName == String.Empty)
+            if (String.IsNullOrEmpty(ownerName))
```

Comment:
No semantic change: ownerName cannot be null - or at least shouldn't ever be null.